### PR TITLE
fix warning: The naming option = 'indicator.scenario' is deprecated...

### PR DESCRIPTION
**Pull request by pascal-sauer**
see https://github.com/pik-piam/mrland/commit/b838c93fb0e603f79499a5754f0d67412e784300 for actual changes, the rest is linter/formatting, but note I removed an unused variable

### CODE REVIEW

File: R/readS4Nproject_input.R [hunk 1]
```diff
@@ -1,90 +1,93 @@
...
     # Disaggregate regional bioenergy to country values using population weight
     # country population in SSP2
-    pop     <- calcOutput("Population", 
-                          naming = "indicator.scenario",
-                          extension2150 = "none",
-                          aggregate = FALSE, 
-                          years = paste0("y", c(2005, seq(2010, 2100, by = 10))), 
-                          round = 6)
-    pop     <- collapseNames(pop[,,"SSP2"])
+    pop     <- collapseNames(calcOutput("Population",
```
Comment (johanneskoch94) [inline R/readS4Nproject_input.R:71]: You can drop "collapseNames" entirely :)

Comment (johanneskoch94) [reply R/readS4Nproject_input.R:71]: Did you run the comparison with `scenario = "SSP2"`?

Comment (johanneskoch94) [reply R/readS4Nproject_input.R:71]: Ok, I see... collapseNames deletes the data name, if only 1 data dimension exists.

File: R/readS4Nproject_input.R [hunk 2]
```diff
@@ -1,90 +1,93 @@
...
+                         stringsAsFactors = FALSE)
+    popReg <- merge(popReg, regionmapping, "Region")
+    popReg <- collapseNames(as.magpie(data.frame(iso = popReg$iso, year = popReg$year, Value = popReg$Value)))
 
-    # country population share
-    pop_shr <- pop/pop_reg
-    
     # bioenergy demand weighted by population
-    x <- x * pop/pop_reg
+    x <- x * pop / popReg
```
Comment (johanneskoch94) [inline R/readS4Nproject_input.R:89]: Just a comment, but can't one simply do ` x <- toolAggregate(x, rel = regionmapping, from = "Region", to = "iso", weight = pop)` to disaggregate x?
(I didn't test this!)

Comment (pascal-sauer) [reply R/readS4Nproject_input.R:89]: Seems reasonable, but I didn't want to change more than necessary (to fix the warning + linter), so I don't need to test more than necessary :wink:

**Pull request merged**